planner-api/openapi.yaml

```diff
@@ -572,6 +585,12 @@ components:
                     minLength: 1
                     maxLength: 200
                     example: "Winter tyres"
+                  amount:
```

Comment:
After these changes, GoalData became identical as the objects in this array. You could replace the type definition with reference to GoalData scheme